wgpu-core/src/device/resource.rs

```diff
@@ -142,6 +139,10 @@ pub struct Device<A: HalApi> {
     #[cfg(feature = "trace")]
     pub(crate) trace: Mutex<Option<trace::Trace>>,
     pub(crate) usage_scopes: UsageScopePool<A>,
+
+    /// Temporary storage, cleared at the start of every call,
+    /// retained only to save allocations.
+    temp_suspected: Mutex<Option<ResourceMaps<A>>>,
```

Comment:
question(blocking): Should we actually even keep this field? The only remaining references to it that I see are setting it to `ResourceMaps::new()`, or clearing it. This makes me question the doc. commentary, too, because we're obviously not re-using anything.

Reply:
Well, that last reference in `Device::untrack` is still meaningful. The code is broken, but the remainder of #5594 that still applies after this PR would make it effective.

Whether it's worthwhile to save that allocation, I have no idea. We have no numbers!